src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -300,7 +314,14 @@ internal sealed class RSession : IRSession, IRCallbacks {
             using (var evaluation = await evaluationSource.Task) {
                 // Load RTVS R package before doing anything in R since the calls
                 // below calls may depend on functions exposed from the RTVS package
-                await LoadRtvsPackage(evaluation);
+                string libPath;
+                if (BrokerConnector.BrokerUri.Scheme == "file") {
+                    libPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().GetAssemblyPath());
```

Comment:
Yes, although how exactly depends on the option that we choose to take.

---
In reply to: [74969930](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74969930) [](ancestors = 74969930)